etc/parameters/default.json

```diff
@@ -110,9 +110,9 @@
       "maximum_distance_to_robot": 6.0,
       "maximum_fit_distance_in_ground": 0.05,
       "maximum_gap_on_line": 0.5,
-      "maximum_merge_gap_in_pixels": 30,
+      "maximum_merge_gap_in_pixels": 2,
       "maximum_number_of_lines": 10,
-      "maximum_projected_segment_length": 0.3,
+      "maximum_projected_segment_length": 1.0,
```

Comment:
Let's discuss these parameter changes. They make the line detection more liberal but maybe also opens room for false positives.

Reply:
Ok 👍🏻